e2e/tests/submissions/author.test.js

```diff
@@ -100,12 +100,6 @@ describe('author submissions', () => {
       linkEl => linkEl.href
     );
 
-    const isSubmitButtonDisabled = await page.$eval(
-      SUBMIT_BUTTON_SELECTOR,
-      submitButton => submitButton.disabled
-    );
-
     expect(authorUpdateLink).toEqual(`${routes.AUTHOR_SUBMISSION}/999108`);
-    expect(isSubmitButtonDisabled).toBe(true);
```

Comment:
would be better if you click the button first, then check if it's disabled, rather than removing this completely.

Reply:
but the button never becomes disabled now. It is always enabled but if there is an error, then it scrolls up